src/extension/kernel.ts

```diff
@@ -832,7 +901,11 @@ export class Kernel implements Disposable {
       'cell.success': successfulCellExecution?.toString(),
       'cell.mimeType': annotations.mimeType,
     })
-    runmeExec.end(successfulCellExecution, Date.now())
+    const endTime = Date.now()
+
+    this.#handleTrackCellRun(cell, successfulCellExecution, startTime, endTime)
```

Comment:
@peraltafederico I think the call should be conditioned to the telemetry flag `isTelemetryEnabled`. Take a look at https://github.com/stateful/vscode-runme/commit/4c11a9c2dbaf8a56e9dcac07febbc5d2f7eae744#diff-2b0a484742f2baf7a6b33fec5aa69dfbf17fbe089363cec3549d0b88f664b1b4R23

Reply:
Fixed in 99e68515e54b2cb2745ea9658518dfc16b5cd32e